src/test/unit/engine/CppRegionTest.cpp

```diff
@@ -110,15 +133,18 @@ void testCppInputOutputAccess(Region *level1) {
   data_actual[12] = 54321;
 }
 
-void testCppLinking(std::string linkPolicy, std::string linkParams) {
+
+
+TEST(CppRegionTest, testCppLinkingFanIn) {
   Network net = Network();
 
   Region *region1 = net.addRegion("region1", "TestNode", "");
   Region *region2 = net.addRegion("region2", "TestNode", "");
-  net.link("region1", "region2", linkPolicy, linkParams);
+
+  net.link("region1", "region2", "TestFanIn2", ""); //the only change testCppLinking* is here
```

Comment:
this is ugly, I had to duplicate the 2 methods, and only this line (link) changes. Didn't know other way how to do it for gtest.